basil/HL/hp_e3632a.yaml

```diff
@@ -0,0 +1,20 @@
+# Device description for HP E3632A Powersupply.
+# set_ function expect a parameter, get_ function return a parameter.
+# Just the very basic commands are imlemented here.
+identifier : HEWLETT-PACKARD,E3632A,0,1.1-5.0-1.0
```

Comment:
The identifier does not have to be fully matched. So sometimes it is better to not specify minor firmware versions for compatibility. I guess _HEWLETT-PACKARD,E3632A_ could be enough here?